packages/wonder-blocks-core/src/util/focus.ts

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
Copied this over to the core package from the [popover utilities](https://github.com/Khan/wonder-blocks/blob/42a238515fa5cbaa1eb460f21819bcb9413223de/packages/wonder-blocks-popover/src/util/util.ts) so that we can use it in the Tabs component.

Created https://khanacademy.atlassian.net/browse/WB-1920 for consolidating the focus utilities into the core package (the modal package also has focus utilities!)